<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>591</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>591</width>
      <height>214</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="categoryLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="payment_dayLabel">
       <property name="text">
        <string>Payment Day</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="payment_dayLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="initial_valueLabel">
       <property name="text">
        <string>Initial Value</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="initial_valueLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="is_active_checkBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Activated?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
